docs/user_guide/emummc/partitioning_sd.md

```diff
@@ -8,12 +8,11 @@
 
 -----
 
-### What you need (Windows)
+## Windows Instructions
 
-!!!tip ""
+!!!tip "What you need (Windows)"
 	- [Minitool partition wizard (or any other good partition manager)](https://www.partitionwizard.com/download/v11/pw11-free.exe)
 
------
 
 ### Instructions (Windows)
```

Comment:
Maybe it's a good idea to make users backup their nintendo folder before we start partitioning?